test/support/scripted_mqtt_server.ex

```diff
@@ -19,19 +19,23 @@ defmodule JackalopeTest.ScriptedMqttServer do
   alias __MODULE__, as: State
 
   # Client API
+  @spec start_link() :: GenServer.on_start()
   def start_link() do
     start_link([])
   end
 
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
   def start_link(opts) do
     GenServer.start_link(__MODULE__, opts)
   end
 
+  @spec enact(atom | pid | {atom, any} | {:via, atom, any}, any) :: any
```

Comment:
```suggestion
  @spec enact(GenServer.server()) :: any
```

Does it really return an `any`?